spark/spark-2.0/pom.xml

```diff
@@ -9,14 +9,13 @@
 		<relativePath>../../pom.xml</relativePath>
 	</parent>
 
-	<artifactId>hivemall-spark</artifactId>
-	<name>Hivemall on Spark</name>
+	<artifactId>hivemall-spark_2.11</artifactId>
```

Comment:
yes, you can use properties in the top-level pom.xml for cross-modules global properties.